README.md

```diff
@@ -0,0 +1,103 @@
+![](https://img.shields.io/badge/microverse-blueviolet)
+
+# Vacation Booking App(Front-end)
+
+> Vacation Booking App is a tool that helps users to plan for thier vacation trips. User can view a list of beautiful destinations, register and choose one or many they want to book a trip for. Regestered users are able to view a list of destinations they booked. Only admin users are allowed to add destinations.
+
+![vacations-booking-screenshot](https://user-images.githubusercontent.com/14215157/143226630-4772855a-9640-4aaf-a595-11be928e8e5a.png)
+
+## Built With
+
+- React
+- Redux
+
+
+## Getting Started
+
+To get a local copy up and run the app, follow these simple example steps.
+
+### Prerequisites
+
+- Node.js > 16.13.0
+- npm > 8.1.3
+- yarn > 1.22.15
+
+### Setup
+
+Clone the repository with:
+
+```
+git clone git@github.com:danmainah/vacations-booking-front-end.git
+```
+or download [ZIP file](https://github.com/danmainah/vacations-booking-front-end/archive/refs/heads/dev.zip)
+
+### Install
+To install all dependencies, run:
+```
+npm install
+```
+
+### Usage
+To open it in the browser, in the project directory, run:
+
+ ```
+ npm start
+ ```
+ Choose port 3001 or any other port number different from 3000 to run your server on it
+ and open `http:localhost:port_number/` on your browser.
+ You should be able to see the home page
```

Comment:
I like this,

[Optional] GENERAL Future ADVISE - Consider when setting up projects that you a have maybe a workflow that allows to test on cloud so everyone sees the current test version of the site as they code and not have to wait to git clone. In the case with this project, they easily test without having to worry about backend running since backend is already running a cloud test environment (for this case could easily be heroku ) that way the front-end only folks do not have to worry about the back-end at all. 😄